@xen-orchestra/xapi/vdi.mjs

```diff
@@ -76,6 +76,11 @@ class Vdi {
       })
     })
 
+    if (gm?.PV_drivers_detected) {
+      // It may happen that `PV_drivers_detected` returns `true` too quickly, and thus we destroy the cloudConfig VDI before the first boot
+      await new Promise(resolve => setTimeout(resolve, 1000 * 30))
```

Comment:
I think it should be a  parameter `minimalRunningDelay` (or something similar) of this method, with a default value of 30 * 1000 . 

ideally the calling code could pull this value from app.config, but we can add this when we'll need